README.md

```diff
@@ -1 +1,5 @@
-# Review-past-project
\ No newline at end of file
+# Review-past-project
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
it would be better to make a more descriptive readme file for your project.

Reply:
- [ ] It is important to use polite words when requesting for changes so it doesn't come off as if you are giving an order. I would suggest you make use of words like `please`, `i think it would`, `i suggest`.
- [ ] please also ensure to state the reason why you are requesting a change, that way the student understands why  the change is requested and is enthusiastic to work on it and improve :+1: